### PR TITLE
Closing times

### PR DESCRIPTION
**Pull request by dev-matan-tsuberi**

### CODE REVIEW

**Comment by dev-matan-tsuberi** [comment]
[Correct closing times for proposals, especially boosted ones](https://trello.com/c/QV7Suz06/92-correct-closing-times-for-proposals-especially-boosted-ones)

File: src/actions/arcActions.ts [hunk 1]
```diff
@@ -186,8 +188,12 @@ export async function getDAOData(avatarAddress: string, currentAccountAddress: s
         transactionState: TransactionStates.Confirmed,
       };
 
+      console.log('boostedPhaseTime', Number(proposalDetails[8]));
```
Comment (tibetsprague) [inline src/actions/arcActions.ts]: remove consoles

File: src/actions/arcActions.ts [hunk 2]
```diff
@@ -508,6 +520,8 @@ export function createProposal(daoAvatarAddress: string, title: string, descript
       const proposal = {
         beneficiary,
         boostedTime: 0,
+        boostedVotePeriodLimit: votingMachineParams[2],
```
Comment (tibetsprague) [inline src/actions/arcActions.ts]: need to cast to Number right?

File: src/actions/arcActions.ts (hunk 1 above)
Comment (dev-matan-tsuberi) [reply src/actions/arcActions.ts]: updated: https://github.com/daostack/alchemy/pull/38/commits/0f0f22d876c6d80100aa110ec022dd755d6c255e

File: src/actions/arcActions.ts (hunk 2 above)
Comment (dev-matan-tsuberi) [reply src/actions/arcActions.ts]: updated: https://github.com/daostack/alchemy/pull/38/commits/0f0f22d876c6d80100aa110ec022dd755d6c255e

**Pull request merged**